requirements/base.txt

```diff
@@ -210,46 +201,42 @@ requests-oauthlib==2.0.0
     # via
     #   commercetools
     #   social-auth-core
-segment-analytics-python==2.3.2
+segment-analytics-python==2.3.3
     # via -r requirements/base.in
 semantic-version==2.10.0
     # via edx-drf-extensions
-simplejson==3.19.2
+simplejson==3.19.3
     # via django-rest-swagger
 six==1.16.0
     # via
     #   edx-auth-backends
     #   edx-django-release-util
     #   python-dateutil
-slumber==0.7.1
-    # via edx-rest-api-client
-social-auth-app-django==5.4.1
+social-auth-app-django==5.4.2
     # via edx-auth-backends
 social-auth-core==4.5.4
     # via
     #   edx-auth-backends
     #   social-auth-app-django
-sqlparse==0.5.0
+sqlparse==0.5.1
     # via django
-stevedore==5.2.0
+stevedore==5.3.0
     # via
     #   edx-django-utils
     #   edx-opaque-keys
-stripe==10.0.0
+stripe==11.2.0
```

Comment:
@attiyaIshaque there is major change in stripe version. Please consult the changelogs to see if there is any breaking change that needs to catered to in our code. Also please keep in mind to test all stripe related functionality on staging too.

Reply:
I noticed the Stripe change, @aht007 . So, I have it in mind that we need to test the Stripe functionality at this stage.

Reply:
We also need to review the Changelogs. Can you consult stripe documentation. It is a major change.